src/main/java/com/musala/atmosphere/client/Device.java

```diff
@@ -1433,14 +1442,197 @@ public Boolean restoreWifiConnectionProperties() {
         return (Boolean) communicator.sendAction(RoutingAction.UNSHAPE_DEVICE);
     }
 
+    public void startLogcat() {
+        startLogcat(LOCAL_DIR, LogCatLevel.VERBOSE);
```

Comment:
It will be good if you add a directory for the LogCat. Don't save it directly in the local directory.